comms/dht/src/store_forward/saf_handler/task.rs

```diff
@@ -341,7 +341,7 @@ where S: Service<DecryptedDhtMessage, Response = (), Error = PipelineError>
 
                     // Every other error shouldn't happen if the sending node is behaving
                     Err(err) => {
-                        //  #banheuristics
+                        // TODO: #banheuristics
```

Comment:
```suggestion
                        //  #banheuristics
```

Created an issue specifically for this: https://github.com/tari-project/tari/issues/5709

Reply:
todo's in code gets left out and we forget about them, a todo with doing is worth creating an issue for :)

Reply:
Thanks working on SAF bans as we speak